js/bleutrade.js

```diff
@@ -306,6 +328,105 @@ module.exports = class bleutrade extends bittrex {
         };
     }
 
+    parseLedgerEntryType (type) {
+        // deposits don't seem to appear in here
+        const types = {
+            'TRADE': 'trade',
+            'WITHDRAW': 'withdrawal',
```

Comment:
Need to change this a bit, for a ledger entry it is `transaction`, not `withdrawal`/`deposit`. The direction indicates whether it's a deposit or a withdrawal, but the type if `transaction, meaning a blockchain thing.

Reply:
Got it. Thanks.